pkg/signup/signup_service_test.go

```diff
@@ -302,3 +286,27 @@ func newSignupServiceWithFakeClient() (signup.Service, *fake.FakeUserSignupClien
 		MasterUserRecords: fakeMURClient,
 	}, fakeClient, fakeMURClient
 }
+
+func (s *TestSignupServiceSuite) newUserSignupComplete() *v1alpha1.UserSignup {
```

Comment:
I would rather have this function without a pointer receiver, so it can be used in other tests, and thus expose the following signature:
```
func newUserSignupComplete() (*v1alpha1.UserSignup, error)
```

Reply:
There is no other tests in this package. And I doubt we would need to use it in other packages. But even if we did then we would need to export the function which would require as to move it to another package, etc...
I would keep it as is since it's simple and clean. Let's move it somewhere else when/if we really need it.